Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -14,6 +14,10 @@ extension OnlinePersistenceService: AnnotationsPersistence {
         await remotePersistence.annotations.deleteAnnotation(annotation: annotation)
     }
 
+    func createOrUpdateAnnotation(annotation: Annotation) -> Annotation? {
```

Comment:
Why are the methods on lines 17 and 21 calling local persistence when it is in the `OnlinePersistenceService+AnnotationsPersistence.swift` file?

Reply:
online persistence service is supposed to call both local and remote, but remote one isn't implemented (and not necessary for our app). so it only calls the local one.